e2e/session-launcher.test.ts

```diff
@@ -1,23 +1,187 @@
 import {
-  createSession,
   deleteSession,
   loginAsAdmin,
   loginAsUser,
   navigateTo,
 } from './test-util';
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+
+const getMenuItem = (page: Page, menuName: string) => {
+  return page.getByRole('link', { name: menuName });
+};
+
+const getCardItemByCardTitle = (page: Page, title: string) => {
+  return page.locator(`.ant-card:has-text("${title}")`);
+};
+
+const getStartSessionButton = (page: Page) => {
+  return page.getByLabel('power_settings_new');
```

Comment:
Label-based selector will need to modify their test code again if their wording changes.